django/core/management/__init__.py

```diff
@@ -362,8 +362,7 @@ def execute(self):
         try:
             settings.INSTALLED_APPS
         except ImproperlyConfigured as exc:
-            self.settings_exception = exc
-        except ImportError as exc:
+            print(exc)
```

Comment:
debug statement left behind

Reply:
Hi, happy new year to you too! I will be able to pick this up in the next few day, thanks for taking a look at it. :)

Reply:
Hi @jacobtylerwalls, I've updated the code and changed it to stderr now.

<img width="804" alt="image" src="https://user-images.githubusercontent.com/10276811/148279598-6ebfbd28-47b1-42db-8056-95410b1884e0.png">